test/sql/domain_field_types.sql

```diff
@@ -0,0 +1,198 @@
+              fieldBigint
+              fieldNumeric
+              fieldReal
+              fieldDoublePrecision
+              fieldBoolean
+              fieldText
+              fieldCharacter
+              fieldCharacterVarying
+              fieldDate
+              fieldTime
+              fieldTimestamp
+            }
+          }
+        }
+      }
+    $$)
+  );
+
+  -- Should probably test all of the filter but this is just a spot check
+  select jsonb_pretty(
+    graphql.resolve($$
+      {
+        domainTestCollection(filter: {fieldInt: {gt: 41}}) {
+          edges {
+            node {
+              id
+              fieldInt
+            }
+          }
+        }
+      }
+      $$)
+  );
+
+  select jsonb_pretty(
+    graphql.resolve($$
+      {
+        domainTestCollection(filter: {fieldBigint: {gt: 0}}) {
+          edges {
+            node {
+              id
+              fieldBigint
+            }
+          }
+        }
+      }
+      $$)
+  );
+
+  select jsonb_pretty(
+    graphql.resolve($$
+      {
+        domainTestCollection(filter: {fieldNumeric: {gt: "0"}}) {
+          edges {
+            node {
+              id
+              fieldNumeric
+            }
+          }
+        }
+      }
+      $$)
+  );
+
+  -- Check that insert mutation resolves the base types
+  select  jsonb_pretty(
+          graphql.resolve($$
+            {
+              __type(name: "DomainTestInsertInput") {
+                inputFields {
+                  name
+                  type {
+                    name
+                    kind
+                    ofType {
+                      name
+                      kind
+                    }
+                  }
+                }
+              }
+            }
+          $$)
+  );
+
+  select jsonb_pretty(
+    graphql.resolve($$
+      mutation newDomainTest {
+        insertIntoDomainTestCollection(objects: [{
+          fieldInt: 42
+          fieldSmallint: 32767
+          fieldBigint: 2165559898978
+          fieldNumeric: "123.45"
+          fieldReal: 3.14
+          fieldDoublePrecision: 3.141592
+          fieldBoolean: true
+          fieldText: "Hello, world!"
+          fieldCharacter: "A"
+          fieldCharacterVarying: "ABCD"
+          fieldDate: "2022-01-01"
+          fieldTime: "12:34:56"
+          fieldTimestamp: "2022-01-01 12:34:56"
+        }]) {
+          records {
+            id
+            fieldInt
+            fieldSmallint
+            fieldBigint
+            fieldNumeric
+            fieldReal
+            fieldDoublePrecision
+            fieldBoolean
+            fieldText
+            fieldCharacter
+            fieldCharacterVarying
+            fieldDate
+            fieldTime
+            fieldTimestamp
+          }
+        }
+      }
+    $$)
+  );
+
+  rollback to savepoint a;
+
+  -- TODO: Check that update mutation resolves the base types
+  -- TODO: Check that delete mutation resolves the base types
+
```

Comment:
Current tests are written for when `resolve_base_type` is true. It's worth adding tests for when `resolve_base_type` is false and for a [computed column that uses a function](https://supabase.github.io/pg_graphql/computed_fields/) and returns a domain type.